tests/Agent/IntegrationTests/IntegrationTests/LogInstrumentation/Log4netMetricsTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using System;
+using System.Collections.Generic;
+using MultiFunctionApplicationHelpers;
+using NewRelic.Agent.IntegrationTestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NewRelic.Agent.IntegrationTests.LogInstrumentation
```

Comment:
So, the instrumentation wrapper project is called "logging", but we already have an integration testing namespace called "Logging", which tests the agent's own logging features.  Personally, it would make more sense for us to call this new test namespace "Logging" since it will match the instrumentation wrapper project name, and rename the other namespace "AgentLogs" or something like that.  Thoughts?

Reply:
Sounds fine to me,